tests/e2e/specs/keplr/keplr-spec.js

```diff
@@ -87,10 +97,13 @@ describe('Keplr', () => {
       });
       cy.addNewTokensFound();
       cy.getTokenAmount('ATOM').then(tokenValue => {
-        expect(tokenValue).to.equal(0);
+        expect(Number.isFinite(tokenValue)).to.be.true;
       });
       cy.getTokenAmount('BLD').then(tokenValue => {
-        expect(tokenValue).to.equal(331);
+        expect(Number.isFinite(tokenValue)).to.be.true;
+      });
+      cy.getTokenAmount('IST').then(tokenValue => {
+        expect(Number.isFinite(tokenValue)).to.be.true;
```

Comment:
revert this so that our tests are more robust

Reply:
It shows different coins on our computers plus what if the contract gets updated and the coin values change? I feel in the tests for workflows, hard coding the value will not help.

Reply:
the tests are designed for the CI not for a personal device. on the CI the value will be exactly the same everytime.
Our tests are (and should be) deterministic i.e the values at a any point of the test should be known exactly.
If the values change then that is a failure on our part and we need to fix the tests or the setup acordingly

Reply:
I agree. Tests should be deterministic. `expect(Number.isFinite(tokenValue)).to.be.true;` is not only deterministic but it also future-proofs the test suite by making it more adaptable to changes in requirements(contract).

Reply:
@frazarshad Thank You for taking the initiative and having the meeting with me to explain the issue going with this approach. Fixed.